src/webob/acceptparse.py

```diff
@@ -901,25 +699,35 @@ def acceptable_offers(self, offers):
         """
         Return the offers that are acceptable according to the header.
 
-        The offers are returned in descending order of preference, where
-        preference is indicated by the qvalue of the media range in the header
-        that best matches the offer.
+        The matched offers are returned in descending order of preference,
+        where preference is indicated by the qvalue of the media range in the
+        header that best matches the offer.
 
         This uses the matching rules described in :rfc:`RFC 7231, section 5.3.2
         <7231#section-5.3.2>`.
 
-        Any offers that cannot be parsed via
-        :meth:`.Accept.parse_offer` will be ignored.
-
-        :param offers: ``iterable`` of ``str`` media types (media types can
-                       include media type parameters) or pre-parsed instances
-                       of :class:`.AcceptOffer`.
-        :return: A list of tuples of the form (media type, qvalue), in
-                 descending order of qvalue. Where two offers have the same
-                 qvalue, they are returned in the same order as their order in
-                 `offers`.
-        """
-        parsed = self.parsed
+        Any offers that cannot be parsed via :meth:`.parse_offer`
+        will be ignored.
```

Comment:
`.parse_offer` is not currently included in the documentation, so this is not linking to anything?